CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file. This project make usage of the [Yii Versioning Strategy](https://github.com/yiisoft/yii2/blob/master/docs/internals/versions.md).
 
+## 1.0.11 (in progress)
++ [#8](https://github.com/luyadev/luya-testsuite/issues/8) Added class/new feature: CmsBlockGroupTestCase
```

Comment:
Tiny adjustment: Add newline after title and mabye change message to: Added new CmsBLockGroupTestCase in order to test block groups. 

or sth. like this